explainer-server/app/controllers/ApiController.scala

```diff
@@ -41,16 +41,23 @@ class ExplainerApiImpl(
   val explainerDB = new ExplainerDB(config)
   val explainerStore = new ExplainerStore(config)
 
-  def publishExplainerToKinesis(explainer: Atom, actionMessage: String, atomPublisher: AtomPublisher) = {
+  def publishExplainerToKinesis(explainer: Atom, actionMessage: String, atomPublisher: AtomPublisher, eventType: EventType = EventType.Update) = {
     if (config.publishToKinesis) {
       val event = ContentAtomEvent(explainer, EventType.Update, DateTime.now.getMillis)
       atomPublisher.publishAtomEvent(event) match {
-        case Success(_) => Logger.info(s"$actionMessage succeeded")
-        case Failure(err) => Logger.error(s"$actionMessage failed", err)
+        case Success(_) => {
+          Logger.info(s"$actionMessage succeeded")
+          "success"
```

Comment:
You can simply rename when importing:
```
import x.y.{Success => PublishSuccess}
```

I'm not really a fan of abbreviations in code too much but that's a personal view.